src/sentry/seer/anomaly_detection/get_historical_anomalies.py

```diff
@@ -74,7 +81,9 @@ def get_historical_anomaly_data_from_seer(
             },
         )
         return None
-    formatted_data = format_historical_data(historical_data, dataset)
+    formatted_data = format_historical_data(
```

Comment:
nit: it'd be nice to be consistent with kwargs or args for `fetch` and `format`.